docs/reference/service_api.yaml

```diff
@@ -0,0 +1,302 @@
+          content:
+            application/json:
+              schema:
+                type: object
+                description: Environments to be inject on the app
+                additionalProperties:
+                  type: string
+        404:
+          description: Service instance does not exist
+        412:
+          description: Service instance not ready
+        default:
+          $ref: '#/components/schemas/Error'
+
+    delete:
+      summary: Unbind App
+      description: |
+        The service endpoint unbinds the app from the service instance
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/BindAppForm'
+      responses:
+        200:
+          description: App successfully unbound from service instance
+        404:
+          description: Service instance does not exist
+        default:
+          $ref: '#/components/schemas/Error'
+  /resources/{name}/status:
+    get:
+      summary: Service Instance Status
+      description: |
+        The service endpoint returns the service instance status
+      parameters:
+        - name: name
+          in: path
+          description: Instance name
+          required: true
+          schema:
+            type: string
+      responses:
+        202:
+          description: Instance not available yet
+        204:
+          description: Instance running
+        default:
+          $ref: '#/components/schemas/Error'
+
+
+# Object definitions          
+components:
+  schemas:
+    Plan:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Plan name
+        description:
+          type: string
+          description: Plan Description
+    InstanceForm:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Instance name
+        plan:
+          type: string
+          description: Plan name
+        team:
+          type: string
+          description: Team name
+        user:
+          type: string
+          description: User name
+        tag:
+          type: array
+          items:
+            type: string
+      required:
+        - name
+        - plan
+        - team
+        - user
+    BindUnitForm:
+      type: object
+      properties:
+        app-host:
+          type: string
+          description: App host
+        app-name:
+          type: string
+          description: App name
+        unit-host:
+          type: string
+          description: Unit address
+      required:
+        - app-host
+        - app-name
+        - unit-host
+    BindAppForm:
+      type: object
+      properties:
+        app-host:
+          type: string
+          description: App host
+        app-name:
+          type: string
+          description: App name
+      required:
+        - app-host
+        - app-name
+    InstanceInfo:
```

Comment:
Taking a quick look at the code it looks like the InstanceInfo object is actually a map[string]string, do we really only accept `label` and `value` keys?

Reply:
Yeah, I got misled by the docs: https://docs.tsuru.io/stable/services/api.html#additional-info-about-an-instance
I'll fix this.